services/ui/serverless.yml

```diff
@@ -190,9 +189,9 @@ resources:
         LoggingConfiguration:
           DestinationBucketName: ${env:LOGGING_BUCKET, ssm:/configuration/${self:custom.stage}/s3/accessLogsBucket, ssm:/configuration/default/s3/accessLogsBucket}
           LogFilePrefix: ${env:LOGGING_BUCKET, ssm:/configuration/${self:custom.stage}/s3/accessLogsPrefix, ssm:/configuration/default/s3/accessLogsPrefix}
-      DeletionPolicy: Delete
     LoggingBucketPolicy:
       Type: "AWS::S3::BucketPolicy"
+      DeletionPolicy: Retain
```

Comment:
Does this contain something stateful? I was hoping we didn't have to retain policy-esque stuff.

Reply:
I don't think we can delete historical logs simply because we migrated.  I'm sure we have a retention policy of logs.

Reply:
oh!  This.  After I deployed, CDK didn't use this bucket... so we need to keep this policy so that this historical log bucket does get a security finding for allowing non-SSL traffic.